src/builder.rs

```diff
@@ -271,6 +275,11 @@ impl<Other: ExtendableThing> ThingBuilder<Other, ToExtend> {
         }
     }
 
+    /// Finalize the set of extensions that must be populated
+    ///
+    /// From this point is not possible to add additional extensions to the builder.
```

Comment:
Should we add that calling this method (I am not even sure if using the term _method_ is right in Rust) unlocks other methods?

Reply:
We should document the markers we use and refer to it maybe?

Reply:
I am not sure what markers you are referring to, it is just a matter of trait bounds.